app/repositories/SessionRepository.scala

```diff
@@ -41,7 +42,9 @@ class SessionRepository @Inject()(
   ) {
 
   def clear(implicit request: Request[_]): Future[Unit] = {
-    cacheRepo.deleteEntity(request)
+    preservingMdc {
+      cacheRepo.deleteEntity(request)
+    }
```

Comment:
are there other invocations of this that need wrapping?

Reply:
I did find some mongo calls that I'd missed but nothing more involving the deleteEntity function